pmemcheck/tests/trans_expl.stderr.exp

```diff
@@ -0,0 +1,20 @@
+Number of stores not made persistent: 4
+Stores not made persistent properly:
+[0]    at 0x........: main (trans_expl.c:36)
+	Address: 0x........	size: 1	state: DIRTY
+[1]    at 0x........: main (trans_expl.c:37)
+	Address: 0x........	size: 2	state: DIRTY
+[2]    at 0x........: main (trans_expl.c:38)
+	Address: 0x........	size: 4	state: DIRTY
+[3]    at 0x........: main (trans_expl.c:43)
+	Address: 0x........	size: 8	state: DIRTY
+Total memory not made persistent: 15
+
+Number of stores made outside of transactions: 3
```

Comment:
This is misleading - the stores were made within the TX, but the objects were not added to TX, prior to making a store.